syntax = "proto3";
package dpi;

message LoadDataRequest {
    string table = 1;
    string file = 2;
    string schema = 3;
}
